test/built-ins/Array/prototype/entries/non-enumerable.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2013 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+  description: >
+      The method should exist on the Array prototype, and it should be writable
+      and configurable, but not enumerable.
+  es6id: 22.1.3.4
+ ---*/
+
+var proto = Array.prototype;
+
+assert(proto.hasOwnProperty('entries'));
```

Comment:
Definitely unnecessary. I've removed that assertion from this file and the similar assertions from files for the property descriptors of the other Array prototype methods.